docs/user-guide/level0.rst

```diff
@@ -6,36 +6,72 @@ Level 0 Data
-+-------------+-----------+--------+------------------------------------------------------------+
-| FLAGS       | 16        | UNIT   | various flags, To be defined                               |
-+-------------+-----------+--------+------------------------------------------------------------+
-| Packet Size | 16        | UINT   | The size of the packet which can be used to determine the  |
-|             |           |        | number of hits included in the packet                      |
-+-------------+-----------+--------+------------------------------------------------------------+
-| Time Stamp  | 48        | UINT   |                                                            |
-+-------------+-----------+--------+------------------------------------------------------------+
-| Checksum    | 16        | UINT   | For data integrity                                         |
-+-------------+-----------+--------+------------------------------------------------------------+
-| Pixel data  |           |        | This field is repeated based on the number of hits detected|
-+-------------+-----------+--------+------------------------------------------------------------+
-
-
-+-------------+-----------+--------+------------------------------------------------------------+
-| Pixel Data  |           |        |                                                            |
-+-------------+-----------+--------+------------------------------------------------------------+
-| Detector id | 16        | UINT   | The detector id for the location of the hit                |
-+-------------+-----------+--------+------------------------------------------------------------+
-| Hit energy  | 16        | UINT   | The ADC value for the energy of the hit                    |
-+-------------+-----------+--------+------------------------------------------------------------+
+Level 0 data are provided in the `FITS file <https://fits.gsfc.nasa.gov/>`__ format.
+For more information on how to read or write FITS file see `astropy.fits <https://docs.astropy.org/en/stable/io/fits/index.html>`__.
+This section describes the organization the level 0 FITS files.
+Level 0 fits files generally include the unconverted data from the raw binary files of ccsds packets.
+The purpose of these files is to provide the raw data from the raw binary files in a more convinient form for analysis.
+It also provides metadata information which summary the data in the file.
+
+Level 0 event files
+-------------------
+
+This file contains the data from all events that triggered the detectors.
+They consist of 3 HDUs including the primary HDU.
+The primary HDU contains no data and is only used for metadata.
+The two other HDUs are named `SCI` and `PKT`.
+`SCI` includes the event data while `PKT` includes the packet data.
+Each data packet may include one of more event therefore there is a one to many relationship between them.
```

Comment:
I am confused by this. Should it say "Each data packet may include one or more events, therefore there is a one to many..."?